### ISSUE TITLE
Disable multicheckbox by 'templateOptions.disabled'

### ISSUE THREAD

**Issue by razalur** [ngx-formly/ngx-formly#1601]
**I'm submitting a ...**
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[x] support request
```

Hello!
I'm trying to enable/disable multicheckbox ('param2') when the value of other checkbox ('param1') changes.

```
  form = new FormGroup({});
  model = {  param1: false, param2: ['second'] };
  fields: FormlyFieldConfig[] = [
    {
      key: 'param1',
      type: 'checkbox',
      templateOptions: {
        label: 'Papam1:',
      },
    },
    {
      key: 'param2',
      type: 'multicheckbox',
      templateOptions: {
        type: 'array',
        label: 'Multicheckbox',
        options: [
          { key: 'first', value: 'first' },
          { key: 'second', value: 'second' },
        ],
      },
      expressionProperties: {
        'templateOptions.disabled': (model: any) => {
          return this.model.param1;
        }
      },
    },
  ];
```
But it does not work. Could you please help me?!

**Comment by aitboudad** [comment]
fixed in `5.1.7`

**Comment by razalur** [comment]
Thank you very much!
